certif/package.json

```diff
@@ -39,7 +39,7 @@
   },
   "devDependencies": {
     "@1024pix/ember-testing-library": "^0.5.0",
-    "@1024pix/pix-ui": "^26.0.0",
+    "@1024pix/pix-ui": "^26.1.0",
```

Comment:
J'ai approve mais en fait j'ai vu après qu'il y a un commit "Update pix-ui to 28.1.0 in certif", pour ne pas avoir cette modification là.

Reply:
Bien vu, corrigé